docs/dictionary/property/HTMLText.lcdoc

```diff
@@ -106,125 +160,54 @@ tag | textSize
-
-Unicode characters whose numeric value is greater than 255 are encoded as "bignum" entities, with a leading ampersand and trailing semicolon. For example, the Japanese character whose numeric value is 12387 is encoded as "#12387;".
-
->*Note:* The <HTMLText> of a <field(keyword)> or <chunk> includes formatting information for the text, but does not include information about the text <properties> of the <field(keyword)> itself. If you use the <HTMLText> <property> to transfer text between <field(object)|fields>, you must make sure that the destination <field(object)|field's> <textFont> and other text <properties> match the settings of the source field, if you want the text in both <field(object)|fields> to look identical.
-
->*Important:* The <HTMLText> <property> uses a tag <control structure|structure> that is HTML-like, but is not completely standard <HTML>, in order to accommodate the full <range> of text styling available in LiveCode. Specifically:
-* The link, box and threedbox tags, as well as the bgColor attribute of the font tag, have been added to accommodate styles that don't exist in standard HTML.
+When <set|setting> the <HTMLText> of a <field(object) to an <HTML>-formatted string, you can use LiveCode color references for 
+the "color" and "bgcolor" attributes. LiveCode translates these into standard HTML-style color specifications.
+
+If a chunk of text includes more than one of the above styles, LiveCode encloses the text in the tags corresponding to each style, 
+from the inside out. For example, if the word "Flail" in a field is underlined and bold, its corresponding 
+<HTMLText> is "&lt;u&gt;&lt;b&gt;Flail&lt;/b&gt;&lt;/u&gt;".
+
+Special characters (whose ASCII value is greater than 127) are encoded as <HTML> entities. Named <HTML> entities are enclosed by a 
+leading ampersand and trailing semicolon; e.g. &amp;agrave; for &agrave; and &amp;euro; for &euro;. LiveCode recognizes all 253 
+entities defined in the XHTML specification at <http://www.w3.org/TR/html4/sgml/entities.html>.
+
+Unicode characters whose numeric value is greater than 255 are encoded as "bignum" entities, with a leading ampersand and trailing 
+semicolon. For example, the Japanese character whose numeric value is 12387 is encoded as "#12387;".
+
+>*Note:* The <HTMLText> of a <field(object)> or <chunk> includes formatting information for the text, but does not include information 
+about the text <properties> of the <field(object)> itself. If you use the <HTMLText> <property> to transfer text between 
+<field(object)|fields>, you must make sure that the destination <field(object)|field's> <textFont> and other text <properties> match 
+the settings of the source field, if you want the text in both <field(object)|fields> to look identical.
+
+>*Important:* The <HTMLText> <property> uses a tag structure that is HTML-like, but is not completely standard <HTML>, 
+in order to accommodate the full <range> of text styling available in LiveCode. Specifically:
```

Comment:
Yet another pre-existing issue - this instance of the word range shouldn't be linked! And I believe it can then be removed from the references.